doc/source/conf.py

```diff
@@ -12,6 +12,33 @@
 import json
 from sphinx_gallery.sorting import FileNameSortKey
 from ansys_sphinx_theme import ansys_favicon, get_version_match, pyansys_logo_black
+from importlib import import_module
+from pprint import pformat
+from docutils.parsers.rst import Directive
+from docutils import nodes
+from sphinx import addnodes
+
+class PrettyPrintDirective(Directive):
+    """Render a constant using pprint.pformat and insert into the document"""
```

Comment:
```suggestion
    """Renders a constant using ``pprint.pformat`` and inserts into the document."""
```
Descriptions for classes should start with a verb ending in "s" or "es".